templates/pgbouncer.yaml.j2

```diff
@@ -12,6 +12,24 @@ stringData: {{ k8s_pgbouncer_environment | to_json }}
 {% else %}
 stringData: {{ k8s_pgbouncer_environment | default({}) | to_json }}
 {% endif %}
+{% if k8s_pgbouncer_selfsigned_certificate_cluster_issuer %}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
```

Comment:
Should we create the Certificate outside of here, so that we don't have to grant additional API permissions to the deploy-account?

Reply:
Good question. @tobiasmcnulty?